.eslintrc.js

```diff
@@ -13,15 +23,18 @@ module.exports = {
     'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
     'max-len': ['warn', { code: 120 }],
     'vue/attribute-hyphenation': 'off',
+    'vue/multi-word-component-names': 'off',
     'vue/no-deprecated-filter': 'warn',
     'vue/no-deprecated-slot-scope-attribute': 'warn',
     'vue/no-deprecated-v-on-native-modifier': 'warn',
     'vue/no-v-for-template-key-on-child': 'warn',
     'vue/no-v-html': 'off',
+    'vue/no-v-model-argument': 'off',
     'vue/v-on-event-hyphenation': 'off',
     '@typescript-eslint/ban-types': 'warn',
     '@typescript-eslint/no-empty-function': 'off',
-    '@typescript-eslint/no-explicit-any': 'off'
+    '@typescript-eslint/no-explicit-any': 'off',
+    '@typescript-eslint/no-var-requires': 'off'
```

Comment:
This is to fix lines 29 and 41 in `StoreCurrentAccount.spec.ts`. It's also for line 1 in `volar.config.js`.
![no var requires storecurrentacc test and volarconfig](https://github.com/bcgov/business-filings-ui/assets/122301442/739ce847-36c8-45e7-a589-72a05e08060a)

Reference: https://dev.to/msoup/fixing-require-statement-not-part-of-import-statementeslinttypescript-eslintno-var-requires-25n3